update readmy and simplify code (#5)
DevExpress-Examples/xaf-how-to-import-data-in-xaf: Readme.md

# How to import data in XAF

This example demonstrates how to import data from an XML-formatted text file, create business objects based on this data, and show the result in an XAF list view.

## Implementation Details

1. Import data from any custom source (we use an XML string in this example).

> 2. Generate the required business objects based on this data using the following API: [Create, Read, Update and Delete Data](https://docs.devexpress.com/eXpressAppFramework/113711/data-manipulation-and-business-logic/create-read-update-and-delete-data)

Reviewer: ```suggestion
2. Based on this data, generate the required business objects using an approach from the following topic: [Create, Read, Update and Delete Data](https://docs.devexpress.com/eXpressAppFramework/113711/data-manipulation-and-business-logic/create-read-update-and-delete-data)
```

> 2. Based on this data, generate the required business objects using an approach from the following topic: [Create, Read, Update and Delete Data](https://docs.devexpress.com/eXpressAppFramework/113711/data-manipulation-and-business-logic/create-read-update-and-delete-data)

Reviewer: ```suggestion
2. Based on this data, generate the required business objects using an approach from the following topic: [Create, Read, Update and Delete Data](https://docs.devexpress.com/eXpressAppFramework/113711/data-manipulation-and-business-logic/create-read-update-and-delete-data).
```

> 2. Based on this data, generate the required business objects using an approach from the following topic: [Create, Read, Update and Delete Data](https://docs.devexpress.com/eXpressAppFramework/113711/data-manipulation-and-business-logic/create-read-update-and-delete-data).

Reviewer 2: ```suggestion
2. Based on this data, generate the business objects using an approach from the following topic: [Create, Read, Update, and Delete Data](https://docs.devexpress.com/eXpressAppFramework/113711/data-manipulation-and-business-logic/create-read-update-and-delete-data).
```
